crosslink/cmd/root.go

```diff
@@ -35,9 +35,18 @@ func newCommandConfig() *commandConfig {
 	c := &commandConfig{
 		runConfig: cl.DefaultRunConfig(),
 	}
+
 	preRunSetup := func(cmd *cobra.Command, args []string) {
 		c.runConfig.ExcludedPaths = transformExclude(c.excludeFlags)
 
+		if c.runConfig.RootPath == "" {
+			cwd, err := os.Getwd()
+			if err != nil {
+				log.Printf("Could not get current working directory: %e", err)
+			}
+			c.runConfig.RootPath = cwd
```

Comment:
Set this as the default value of the flag instead so that it is communicated to users.